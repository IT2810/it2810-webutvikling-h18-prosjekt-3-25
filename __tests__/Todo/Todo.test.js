import React from 'react';
import TodoScreen from '../../components/StepCounter.js';
import renderer from 'react-test-renderer';
import ShallowRenderer from 'react-test-renderer/shallow';

it("renders correctly", () => {
    const tree = renderer.create(<TodoScreen />).toJSON();
    expect(tree).toMatchSnapshot();
})
