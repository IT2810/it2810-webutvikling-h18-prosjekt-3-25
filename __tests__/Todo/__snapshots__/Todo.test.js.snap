// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <View
    style={
      Object {
        "marginTop": "10%",
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {},
          undefined,
          undefined,
          Object {
            "fontSize": 35,
          },
          undefined,
          undefined,
          undefined,
          Object {},
          undefined,
          undefined,
          Object {
            "alignSelf": "center",
          },
        ]
      }
    >
      Daily goal
    </Text>
    <View
      animationType="timing"
      disabled={true}
      onLayout={[Function]}
      step={0}
      style={
        Array [
          Object {
            "height": 40,
            "justifyContent": "center",
          },
          false,
          Object {
            "alignSelf": "center",
            "width": "80%",
          },
        ]
      }
      thumbTouchSize={
        Object {
          "height": 40,
          "width": 40,
        }
      }
      value={0}
    >
      <View
        onLayout={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "#b3b3b3",
            },
            Object {
              "borderRadius": 2,
              "height": 4,
            },
            undefined,
          ]
        }
      />
      <View
        style={
          Object {
            "backgroundColor": "#3f3f3f",
            "borderRadius": 2,
            "height": 4,
            "marginTop": -0,
            "opacity": 0,
            "position": "absolute",
            "width": 0,
          }
        }
      />
      <View
        onLayout={[Function]}
        style={
          Object {
            "backgroundColor": "red",
            "borderRadius": 10,
            "height": 20,
            "opacity": 0,
            "position": "absolute",
            "top": 22,
            "transform": Array [
              Object {
                "translateX": 0,
              },
              Object {
                "translateY": -0,
              },
            ],
            "width": 20,
          }
        }
      />
      <View
        onMoveShouldSetResponder={[Function]}
        onMoveShouldSetResponderCapture={[Function]}
        onResponderEnd={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderReject={[Function]}
        onResponderRelease={[Function]}
        onResponderStart={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        onStartShouldSetResponderCapture={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
              "bottom": 0,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            },
            Object {},
          ]
        }
      />
    </View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {},
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          Object {
            "alignSelf": "center",
          },
        ]
      }
    >
      Can't reach the pedometer, please refresh the page
    </Text>
  </View>
  <View
    style={
      Object {
        "marginTop": "10%",
      }
    }
  >
    <View
      style={
        Array [
          Object {},
          undefined,
        ]
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {},
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            Array [
              Object {
                "color": "#86939e",
                "fontSize": 15,
                "fontWeight": "bold",
                "marginBottom": 1,
                "marginLeft": 20,
                "marginRight": 20,
                "marginTop": 15,
              },
              undefined,
              undefined,
            ],
          ]
        }
      >
        Set daily goal
      </Text>
    </View>
    <View
      style={
        Object {
          "borderBottomColor": "#bdc6cf",
          "borderBottomWidth": 1,
          "marginLeft": 20,
          "marginRight": 20,
          "transform": Array [
            Object {
              "translateX": 0,
            },
          ],
        }
      }
    >
      <TextInput
        allowFontScaling={true}
        onChangeText={[Function]}
        onSubmitEditing={[Function]}
        placeholder="Eg. 8000"
        style={
          Array [
            Object {
              "color": "#86939e",
              "minHeight": 36,
              "width": 750,
            },
            Object {
              "fontSize": 17.5,
            },
            undefined,
          ]
        }
        value=""
      />
    </View>
    <View
      style={
        Array [
          Object {},
          undefined,
        ]
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {},
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            Array [
              Object {
                "color": "#ff190c",
                "fontSize": 15,
                "marginBottom": 1,
                "marginLeft": 20,
                "marginRight": 20,
                "marginTop": 5,
              },
              undefined,
              undefined,
            ],
          ]
        }
      >
        
      </Text>
    </View>
    <View
      style={
        Array [
          Object {
            "marginLeft": 15,
            "marginRight": 15,
          },
          undefined,
          undefined,
          undefined,
        ]
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={null}
      >
        <View
          pointerEvents="box-only"
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "#9E9E9E",
                "flexDirection": "row",
                "justifyContent": "center",
                "padding": 19,
              },
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              Object {
                "padding": 12,
              },
              undefined,
              undefined,
              undefined,
              Object {
                "backgroundColor": "#4286f4",
              },
              undefined,
              undefined,
            ]
          }
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Array [
                Object {},
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                Array [
                  Object {
                    "color": "white",
                    "fontSize": 20,
                  },
                  undefined,
                  Object {
                    "fontSize": 17.5,
                  },
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                ],
              ]
            }
          >
            Ok
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;
